f2/src/forms/LocationInfoForm.js

```diff
@@ -103,6 +104,7 @@ export const LocationInfoForm = ({ onSubmit }) => {
             <Field
               name="postalCode"
               label={<Trans id="locationinfoPage.postalCode" />}
+              errorMessage={<Trans id="locationInfoForm.labelWarning" />}
```

Comment:
simply use `locationInfoForm.warning` id! You can reuse the same translation.

Reply:
Also, this means you can remove lines 54 to 58! the `{false ? ( // mark ids for lingui` section

Reply:
fixed. Thank you